aws-alb-ingress-controller/Makefile

```diff
@@ -1,9 +1,23 @@
+KUBERNETES_VERSION="1.18.1"
 RELEASE = "$$(basename $$PWD)"
 
 .PHONY: install
 install:
 	helm upgrade -i --wait --set readinessProbe.enable=false --set livenessProbe.enable=false $(RELEASE) .
 
+.PHONY: lint
+lint: lint-default
+
+.PHONY: lint-default
+lint-default:
+	@echo "=> Indent check default values.yaml"
+	../.circleci/lint-chart.sh $$PWD/templates
+	@echo "=> Linting default values.yaml"
+	helm lint --strict
```

Comment:
This lint depends on the version of helm
https://github.com/helm/helm/blob/fee2257e3493e9d06ca6caa4be7ef7660842cbdb/pkg/lint/rules/deprecations.go